src/Tizen.NUI/src/public/Application/NUIApplication.cs

```diff
@@ -158,17 +158,14 @@ public NUIApplication(Size2D windowSize, Position2D windowPosition, ThemeOptions
         }
 
         /// <summary>
-        /// The constructor with a stylesheet, window mode, window size, position and default window type.
+        /// The constructor with a stylesheet, window mode and default window type.
+        /// Especially, It is for IME window.
         /// </summary>
         /// <param name="styleSheet">The styleSheet URL.</param>
         /// <param name="windowMode">The windowMode.</param>
-        /// <param name="windowSize">The window size.</param>
-        /// <param name="windowPosition">The window position.</param>
         /// <param name="type">The default window type.</param>
```

Comment:
should we remove windowSize and windowPosition?

Reply:
I think.. it is best way.
If the size/position is required, I will add new API with size/position.